### DISCUSSION TITLE
LangSmith Proxy - is it possible to use a project aside from default?

### DISCUSSION THREAD

**Discussion by andrewmatchday** [langchain-ai/langsmith-sdk#616] [Q&A] [answered]
I recently tried the LangSmith Proxy and it works great but I noticed that if you're tracing via the REST API you can specify the project to store the runs by setting `session_name`. All my tracing runs via the proxy showed up in the `default` project so I was wondering if it's possible to change that? Also, is it be possible to include other additional custom metadata in the trace runs while we're using the proxy?

**Comment by hinthornw** [comment, marked as answer]
If you start the proxy with the env var set (LANGCHAIN_PROJECT) it will route there. There isn't currently support for dynamic project selectiong
